src/modules/redesign/i18n/fr_FR/awards.json

```diff
@@ -0,0 +1,4 @@
+{
+	"gold": "You have reached gold!",
+	"title": "Own awards"
+}
```

Comment:
translation?

Reply:
I have created all the files.
I translate them, sorry
Translation ok